### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by honeyakshat999**
just corrected some grammatical mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,11 +83,11 @@ Installation Instructions
 - *The hard way (automatic)*: `make copy-install` will copy the bin files into /usr/local/bin (uninstall with `make usrlocal-uninstall`).
 - *The symlinks way*: `make symlinks-install` will make symlinks of the bin files into /usr/local/bin (uninstall with `make usrlocal-uninstall`) (useful when doing development)
 - *The MacPorts Way*: `sudo port install vcs_blackbox`
-- *The Homebrew Way*: `brew install blackbox`
```
Comment (TomOnTime) [inline README.md]: "install blackbox" is correct. Remove the space.

File: README.md [hunk 2]
```diff
@@ -253,7 +253,7 @@ Small strings, such as passwords and API keys, are stored in a hiera yaml file,
   - ...
 ```
 
-In blackbox.yaml specify:
+In Blackbox.yaml specify:
```
Comment (TomOnTime) [inline README.md]: Leave the "B" lowercase.

File: README.md [hunk 3]
```diff
@@ -428,7 +428,7 @@ Where is the configuration stored? .blackbox vs. keyrings/live
 ==============================================================
 
 Blackbox stores its configuration data in the `.blackbox` subdirectory.  Older
-repos use `keyrings/live`.  For backwards compatibility either will work.
+repo's use `keyrings/live`.  For backward compatibility either will work.
```
Comment (TomOnTime) [inline README.md]: Leave the "B" lowercase.

File: README.md [hunk 4]
```diff
@@ -557,7 +557,7 @@ Save the passphrase somewhere safe!
 Create a sub-key that has no password:
 
 ```
-$ gpg --homedir . --edit-key svc_deployacct
+$ gpg -- homedir. --edit-key svc_deployacct
```
Comment (TomOnTime) [inline README.md]: Remove the space between `--` and `homedir`

**Comment by tlimoncelli** [comment]
This has become stale.  Most of the fixes are implemented in other PRs.  Closing.

**Pull request closed without merging**